src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -108,6 +123,11 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
         <br/>
         <button type="submit">Send</button>
     </form>
+    <form action ="<%= blobstoreService.createUploadUrl("/ImageUploadServlet") %>" method= "POST" enctype = "multipart/form-data">
```

Comment:
The " maybe be messing this line up. It might be fixed by changing the " -> ' around /ImageUploadServlet